capstone/src/main/webapp/document_analyzer.html

```diff
@@ -30,64 +31,69 @@ <h3 class="masthead-brand">Perspective Hub</h3>
+              <label class="form-check-label" for="byWord">By word</label>
+            </div>
+            <div class="form-check form-check-inline">
+              <input class="form-check-input" type="radio" name="analysisRadios" id="noAnalysis" value="NONE">
+              <label class="form-check-label" for="noAnalysis">No analysis</label>
+            </div>
+          </div>
+          <button class="btn btn-primary" id="submit_button" style="display: none;" type="submit">Submit</button>
+          <button class="btn btn-secondary" id="authorize_button" style="display: none;">Authorize</button>
+          <button class="btn btn-secondary" id="signout_button" style="display: none;">Sign Out</button>
+        </form>
+        <div id="separator-container"></div>
+        <div id="chart-container"></div>
+        <div id="analysis-container"></div>
       </div>
-      <small id="disclaimer" class="form-text text-muted">Data may be stored for QA/QC purposes</small><br>
-      <form>
-        <div class="form-check">
-          <input type="checkbox" class="form-check-input" id="toxicity">
-          <label class="form-check-label" for="toxicity">Toxicity</label>
-        </div>
-        <div class="form-check">
-          <input type="checkbox" class="form-check-input" id="identity-attack">
-          <label class="form-check-label" for="identity-attack">Identity Attack</label>
-        </div>
-        <div class="form-check">
-           <input type="checkbox" class="form-check-input" id="insult">
-          <label class="form-check-label" for="insult">Insult</label>
-        </div>
-        <div class="form-check">
-          <input type="checkbox" class="form-check-input" id="profanity">
-          <label class="form-check-label" for="profanity">Profanity</label>
-        </div>
-        <div class="form-check">
-          <input type="checkbox" class="form-check-input" id="threat">
-          <label class="form-check-label" for="threat">Threat</label>
-        </div>
-        <div class="form-check">
-          <input type="checkbox" class="form-check-input" id="sexually-explicit">
-          <label class="form-check-label" for="sexually-explicit">Sexually Explicit</label>
-        </div>
-        <div class="form-check">
-          <input type="checkbox" class="form-check-input" id="flirtation">
-          <label class="form-check-label" for="flirtation">Flirtation</label>
-        </div><br>
-        <button type="submit" class="btn btn-primary">Submit</button>
-      </form>
-      <div id="perspective-response"></div>
       <footer class="mastfoot mt-auto">
         <div class="inner">
-          <p>Built by Christopher Mohri, Daniel Ortega and Oluwadamilare Alamutu</p>
+          <p>Built by Christopher Mohri, Daniel Ortega and Oluwadamilare Alamutu.</p>
         </div>
       </footer>
     </div>
+    <script src="oAuth.js"></script>
+    <script async defer src="https://apis.google.com/js/api.js"
```

Comment:
These callbacks are a little confusing.  Is it possible to have the call to `handleClientLoad` (which I might suggest calling something like `initOAuth`) in just the `onload` callback?  If not can you add a comment explaining this?

Reply:
This was the starter code in the API documentation - I'll do some research into the callbacks now and see if I can change it.

Reply:
I can't say I know entirely how it works, but it looks like the code was set up to run `handleClientLoad` exactly once, no matter how a browser loads the page. [This article](https://stackoverflow.com/questions/48173424/script-async-defer-onload-usage-in-googles-api-javascript-samples) is the best explanation I could find. I renamed the function to `initOAuth`, but I think it would be best if I leave everything else as is, so as not to remove any edge cases it covers.